pkgs/development/ocaml-modules/uuidm/default.nix

```diff
@@ -0,0 +1,28 @@
+{stdenv, fetchurl, ocaml, findlib}:
+
+stdenv.mkDerivation rec {
+  version = "0.9.5";
+  name = "uuidm-${version}"; 
+  src = fetchurl {
+    url = " http://erratique.ch/software/uuidm/releases/uuidm-${version}.tbz";
+    sha256 = "03bgxs119bphv9ggg97nsl5m61s43ixgby05hhggv16iadx9zndm";
+  };
+
+  unpackCmd = "tar -xf $curSrc";
+
+  buildInputs = [ocaml findlib];
+
+  configurePhase = "ocaml setup.ml -configure --destdir $prefix";
+  buildPhase = "ocaml setup.ml -build";
+  installPhase = "ocaml setup.ml -install";
+
+  createFindlibDestdir = true;
+
+  meta = with stdenv.lib; {
+    description = "Uuidm is an OCaml module implementing 128 bits universally unique identifiers version 3, 5 (name based with MD5, SHA-1 hashing) and 4 (random based) according to RFC 4122.";
```

Comment:
Please conform to the recommendations given in the [manual](http://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes).